src/siteconfig/forms.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, *args, **kwargs):
             self.fields['custom_stylesheet'].disabled = True
             self.fields['custom_javascript'].disabled = True
             self.fields['deck_owner'].disabled = True
+            self.fields['enable_shared_library'].disabled = True
```

Comment:
Let's tidy this up a bit by putting these fields in a list at the top of the Form class:
`advanced_fields = ['custom_stylesheet', ...]`

Then loop through them here, and also in the Layout helper advanced accordion section (anywhere else? maybe can add to keys to delete in the test instead of hardcode?)

Reply:
done. added list of deck owner only fields to the META class, instead of hardcoding said fields in `__init__` and the form helper.
```
deck_owner_only = [
            "custom_stylesheet",
            "custom_javascript",
            "deck_owner",
            "enable_shared_library",
        ]
```